Model/Tax/Sales/Total/Quote/Tax.php

```diff
@@ -328,6 +344,23 @@ public function mapItem(
             }
         }
 
+        try {
+            $product = $this->productRepository->getById($item->getProductId(), false, $item->getStore());
+
+            // Configurable products should use the PTC of the child (when available)
+            if ($product->getTypeId() == 'configurable') {
+                $children = $item->getChildren();
+
+                if (is_array($children) && isset($children[0])) {
+                    $product = $this->productRepository->getById($children[0]->getProductId(), false, $item->getStore());
+                }
+            }
+
```

Comment:
Is there a scenario where we would want to use the parent product PTC instead of the child? For example if there is a TjPtc set on the parent but none set on the child? That way it would allow a single categorization to apply to all the child products. I'm thinking of scenarios where the child products are just variations on the parent, say for example T shirts, where the child products are just changes to size or color. In this case all would have the same PTC and the PTC wouldn't have to be applied individually.

Reply:
> Is there a scenario where we would want to use the parent product PTC instead of the child

No.  If a child PTC is defined, it supersedes the parent.  

> if there is a TjPtc set on the parent but none set on the child

The parent PTC is loaded via the Tax Class, but is only used when the child has no PTC.